tests/integration/test_requirements_dev.py

```
@@ -0,0 +1,318 @@
+    """Test the specific changes made in the diff."""
+    
+    @pytest.fixture
+    def requirements(self) -> List[Tuple[str, str]]:
+        """Parse and return requirements."""
+        return parse_requirements(REQUIREMENTS_FILE)
+    
+    def test_pyyaml_added(self, requirements: List[Tuple[str, str]]):
+        """Test that PyYAML was added as per the diff."""
+        pyyaml_entries = [(pkg, ver) for pkg, ver in requirements if pkg == 'PyYAML']
+        assert len(pyyaml_entries) == 1
+        pkg, ver = pyyaml_entries[0]
```

Ruff (0.14.7)

Unpacked variable `pkg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)